tests/screenshots/build.rs

```diff
@@ -37,6 +37,21 @@ pub fn collect_test_cases() -> std::io::Result<Vec<test_driver_lib::TestCase>> {
 }
 
 fn main() -> std::io::Result<()> {
+    let default_font_path: std::path::PathBuf = [
+        env!("CARGO_MANIFEST_DIR"),
+        "..",
+        "..",
+        "examples",
+        "printerdemo",
+        "ui",
+        "fonts",
+        "NotoSans-Regular.ttf",
+    ]
+    .iter()
+    .collect();
+
+    std::env::set_var("SLINT_DEFAULT_FONT", default_font_path);
```

Comment:
Instead of an env variable, we could add that to the CompilerConfiguration  (the private one,  in i_slint_compiler )
It could default to the env variable.  But that's slightly cleaner in the test.  
But that's also not a big difference so feel free to ignore.

Reply:
Yeah, I was thinking about using the API as well, but for the run-time support we need the env variable anyway, so I think it's okay to use it for the compiler, too.